pkg/network/multus/annotation.go

```diff
@@ -22,6 +22,7 @@ package multus
 import (
 	"encoding/json"
 	"fmt"
+	"maps"
```

Comment:
Commit message: Please consider framing the issue in the eyes of potential integrations:
External entities may want to add information to existing NetworkSelectionElements defined by KubeVirt in the `k8s.v1.cni.cncf.io/networks` annotation (For example ipam-extensions).

Please also write about KubeVirt owned fields and what happens to them.
I would also elaborate on the work done WRT the preservation of fields that do not exist in the NetworkSelectionElement struct known to KubeVirt.

nit: There are two "Signed-off-by: Ananya Banerjee <anbanerj@redhat.com>"